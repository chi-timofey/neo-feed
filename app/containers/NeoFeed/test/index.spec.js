import React from 'react';
import renderer from 'react-test-renderer';
import { AsteroidsContainer } from '../';

jest.mock('../../../components/AsteroidsList', () => 'AsteroidsList');

describe('<AsteroidsContainer />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<AsteroidsContainer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
