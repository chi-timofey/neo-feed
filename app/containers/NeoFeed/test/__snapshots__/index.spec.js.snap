// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AsteroidsContainer /> renders correctly 1`] = `
<div
  className="container"
>
  <AsteroidsList />
</div>
`;
